client/my-sites/invites/invite-accept/index.jsx

```diff
@@ -46,6 +46,11 @@ export default React.createClass( {
 	refreshInvite() {
 		const invite = InvitesStore.getInvite( this.props.site_id, this.props.invitation_key );
 		const error = InvitesStore.getInviteError( this.props.site_id, this.props.invitation_key );
+		// add subscription-related keys to the invite
+		Object.assign( invite.invite, {
+			activationKey: this.props.activation_key,
+			authKey: this.props.auth_key
+		} );
```

Comment:
I was wondering if you had change the endpoint to add the `activationKey`. But, now I see where you added it :smile: